site/en/blog/permissions-chip/index.md

```diff
@@ -68,16 +65,17 @@ how Windows users reacted to the geolocation and the notification prompts on sit
 Given an ignore/dismiss rate of approximately 85%, and especially given how much the prompt stands out and
 insists on users making a decision immediately, there is a conflict between the level of urgency
 assumed by the browser versus the user's preference for waiting to make a decision. This creates the
-perception that it's "annoying" for a site to ask for a permission.
+perception that it's "annoying" for a site to ask for a permission, apart from potential extra
+things users need to react to like cookie consent banners, newsletter signups, etc.
```

Comment:
```suggestion
perception that it's "annoying" for a site to ask for a permission because it will be lost in the potential extra
things users need to react to like cookie consent banners, newsletter signups, etc.
```